generation/adapters/android.js

```diff
@@ -2,15 +2,17 @@ const t = require('babel-types');
 const generator = require('../core/generator');
 const { callGlobal } = require('../helpers');
 
-const { isNumber, isString, isBoolean, isOfClass, isArray } = require('../core/type-checks');
+const { isNumber, isString, isBoolean, isArray, isDefined } = require('../core/type-checks');
 
 const typeCheckInterfaces = {
-  Integer: isNumber,
+  'ArrayList<String>': isArray,
+  'Matcher<View>': null, // isOfClass('Matcher'),
```

Comment:
@DanielMSchmidt , maybe let's put `isDefined` check here? Or there are special cases when the matcher is undefined?